examples/react-template/screens/Column.tsx

```
@@ -19,61 +19,121 @@ import {
+            <Text>Size=2</Text>
+          </Box></ColumnsItem>
+      </Columns>
 
+      <Divider/>
+      <Title level={TitleLevels.THREE}>Multiline</Title>
+      <Text> Columns is multiline </Text>
+      <Columns multiline >
+        {
+          Object.values([1,2,3,4,5,6,7,8,9,10,11,12]).map( (colSize, index) => {
+          return (
+            <ColumnsItem size={colSize}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)